tests/lib/tools/simpleget

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os.path
+import argparse
+import logging
+import datetime
+from urllib import request
+from urllib.parse import urlparse
```

Comment:
Since we're using urllib, is there a reason why not to use `urllib.request.urlretrieve`? Then the code would just be:
```python
from urllib import request
request.urlretrieve(opts.URL, opts.output)
```